src/Squidex.Infrastructure.GetEventStore/EventSourcing/GetEventStore.cs

```diff
@@ -54,9 +54,30 @@ public IEventSubscription CreateSubscription(IEventSubscriber subscriber, string
             return new GetEventStoreSubscription(connection, subscriber, projectionHost, prefix, position, streamFilter);
         }
 
-        public Task GetEventsAsync(Func<StoredEvent, Task> callback, CancellationToken cancellationToken, string streamFilter = null, string position = null)
+        public async Task GetEventsAsync(Func<StoredEvent, Task> callback, CancellationToken cancellationToken, string streamFilter = null, string position = null)
         {
-            throw new NotSupportedException();
+            var streamName = await connection.CreateProjectionAsync(projectionHost, prefix, streamFilter);
+
+            var sliceStart = ProjectionHelper.ParsePosition(position) ?? -1;
```

Comment:
this should be
```cs
var sliceStart = ProjectionHelper.ParsePosition(position) ?? 0;
```
Otherwise it throws an exception `System.ArgumentOutOfRangeException: start should be non-negative.` when the position was null.

After changing this I ended up with the following exception:
`System.InvalidOperationException: EventStoreConnection 'ES-52bd9eec-4649-4669-b352-bcc651
cf1686' is not active.`

Reply:
Thats strange

Reply:
If my ES version is 4.0.1.0, would it cause problems?

Reply:
Found the problem: see issue #215

Reply:
Can you help me to test this PR?